wire-ios/Wire-iOS/Sources/UserInterface/Components/Views/MarkdownTextView.swift

```diff
@@ -698,6 +698,23 @@ extension DownStyle {
         style.h3Size = style.h1Size
         return style
     }()
+
+    /// The style used during the login flow
+    static var login: DownStyle = {
```

Comment:
Is this style used for the copy related to the link? If so we can rename it to:

```suggestion
    /// The style used during the login flow
    static var loginExternalLinkStyle: DownStyle = {
```

Reply:
It's not only related to links it the style used in the login flow. Paragraph styling and so on.

Reply:
Then no need for change I guess. @typfel